__tests__/app.test.js

```diff
@@ -87,23 +88,52 @@ describe("GET /api/articles/:article_id", () => {
 });
 
 describe("GET /api/articles", () => {
-  test("200- should respond with an articles array containing article objects, each with the correct properties", () => {
+  test("200- should respond with an array of article objects, each with the correct properties", () => {
     return request(app)
       .get("/api/articles")
       .expect(200)
-      .then(({ body }) => {
-        console.log(body);
+      .then(({ body: { articles } }) => {
+        if (articles.length > 1) {
+          articles.forEach((article) => {
+            expect(article).toHaveProperty("article_id");
+            expect(article).toHaveProperty("title");
+            expect(article).toHaveProperty("topic");
+            expect(article).toHaveProperty("author");
```

Comment:
I would like to see the type of the values as well as the properties  eg title will be a string etc

Reply:
Noted!